serializations.md

```diff
@@ -878,6 +910,38 @@ The binary is a serialized Merkle Patricia Tree.
 ]
 ```
 
+### Generalized accounts
+Generalized accounts are available from version 3, Fortuna release
+
+#### Generalized accounts attach transaction
```

Comment:
Is this orthogonal to the "Not implemented yet" attach in contract_transactions.md? Shall that doc be updated?

Reply:
That one I've simply missed. I think it should be removed, that is the wrong place for it...

Reply:
I see that one detailed in new commit. Thanks.